src/PureScript/CST/Traversal.purs

```diff
@@ -186,7 +186,8 @@ traverseModuleBody
    . Applicative f
   => { | OnBinder (Rewrite e f) + OnDecl (Rewrite e f) + OnExpr (Rewrite e f) + OnType (Rewrite e f) + r }
   -> Rewrite e f ModuleBody
-traverseModuleBody k = Newtype.traverse ModuleBody (\b -> b { decls = _ } <$> traverse k.onDecl b.decls)
+traverseModuleBody k (ModuleBody b) =
+  ModuleBody <<< b { decls = _ } <$> traverse k.onDecl b.decls
```

Comment:
It's no longer possible to use `Newtype.traverse`, due to a failed `Coercible` constraint.